android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/cardano/SignerTestsCardano.kt

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+package com.trustwallet.core.app.blockchains.cardano
+
+import com.google.protobuf.ByteString
+import com.trustwallet.core.app.utils.toHex
+import com.trustwallet.core.app.utils.toHexBytes
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import wallet.core.jni.CardanoSigner
+import wallet.core.jni.proto.Cardano
+
+class TestCardanoSigner {
+
+    init {
+        System.loadLibrary("TrustWalletCore")
+    }
+
+    @Test
+    fun testSignTransaction() {
+        val signingInput = Cardano.SigningInput.newBuilder()
+        signingInput.amount = 1000000
+        signingInput.toAddress = "Ae2tdPwUPEZ4V8WWZsGRnaszaeFnTs3NKvFP2xRse56EPMDabmJAJgrWibp"
+        signingInput.changeAddress = "Ae2tdPwUPEZLKW7531GsfhQC1bNSTKTZr4NcAymSgkaDJHZAwoBk75ATZyW"
+        val outPoint = Cardano.OutPoint.newBuilder()
+            .setTxid(ByteString.copyFrom("59991b7aa2d09961f979afddcd9571ff1c637a1bc0dab09a7233f078d17dac14".toHexBytes()))
+            .setIndex(6)
+            .build()
+        val utxo1 = Cardano.UnspentTransaction.newBuilder()
+            .setOutPoint(outPoint)
+            .setAmount(15000000)
+            .setAddress("Ae2tdPwUPEZ6SqAETdiJgPYHpAey2MWakEVRDESWYzBePi7u5uAL5ah26qx")
+            .build()
+        signingInput.addUtxo(utxo1)
+        val privateKey = "b8c31abcc41d931ae881be11da9e4d9242b1f01cae4e69fa29d5ba1f89f9c1549ec844c6b39c70fa6d3a254fe57c1efee1a75eb9755e0b751e96dd288deabc881ae60957699bf72b212ca823520cf7d86af5d1304cd90248fe60bd1fe442870f".toHexBytes()
+        signingInput.addPrivateKey(ByteString.copyFrom(privateKey))
+
+        // plan
```

Comment:
is this commend needed here? same for sign

Reply:
ok